src/stravalib/model.py

```diff
@@ -687,6 +690,7 @@ class SegmentExplorerResult(
     # Undocumented attributes:
     starred: Optional[bool] = None
 
+    # Could we avoid this step and just assign variables in the validator? the validator
```

Comment:
i was just wondering here why we explicitly call the validator vs just adding a validator to the property. perhaps we can't do that? 

i also was confused as to why this is a property at all vs a list of values. again i'm certain i'm missing something. i've used the .lat attr before i think.